test/bib-marc-test.js

```diff
@@ -1036,6 +1065,7 @@ describe('Bib Marc Mapping', function () {
           // We have one 245 with $6=880-01
           // The first 880 has a valid reverse link in $6
           // but does not define any other subfields, so no valid value:
+          console.log('nyplParallelTitle: ', bib.statement('nypl:parallelTitle'))
           assert(!bib.statement('nypl:parallelTitle'))
```

Comment:
This test asserts that there are no `nypl:parallelTitle` statements, when in fact there is one because you've changed the policy (no longer removing trailing empty values). So this test as is will fail unless we start trimming blank statements again (I propose a different, better place to do that above..)